esphome/components/esp32_camera/__init__.py

```diff
@@ -2,8 +2,11 @@
 import esphome.config_validation as cv
 from esphome import pins
 from esphome.const import (
+    CONF_DISABLED_BY_DEFAULT,
     CONF_FREQUENCY,
     CONF_ID,
+    CONF_INTERNAL,
+    CONF_NAME,
```

Comment:
Not needed after other changes
```suggestion
    CONF_FREQUENCY,
    CONF_ID,
```